src/library_webgl2.js

```diff
@@ -244,13 +247,15 @@ var LibraryWebGL2 = {
       GL.queries[id] = null;
     }
   },
+  glDeleteQueriesEXT: 'glDeleteQueries',
 
   glIsQuery__sig: 'ii',
   glIsQuery: function(id) {
     var query = GL.queries[id];
     if (!query) return 0;
     return GLctx['isQuery'](query);
```

Comment:
For USE_WEBGL2 + webgl1, this and others in this file wouldn't be present.

Reply:
Actually, this file probably wouldn't even be included. So you'll need to instead move the definitions into library_webgl.js, and those definitions can choose at runtime between webgl1+ext and webgl2.

Reply:
I'm no longer doing any EXT aliases on WebGL 2 anymore, so this is resolved as well.